### ISSUE TITLE
When should #include <iomanip> be used? 

### ISSUE THREAD

**Issue by Cherry-Crane** [haomateach/cpp#118]
I saw this in one classmate's assignment 8, but I didn't use it in my code and it is woking well. Dose anyone know when to use it?

**Comment by gavingro** [comment]
I think that <iomanip> is a library that lets you manipulate the format of the things you stream in and out with `cin` and `cout`. 

There was one example in the Udemy from ages ago where Frank used the following code to set it so that any float/decimal value would be rounded (or maybe just only display) the first two decimals, so the output could look like proper prices.
`std::cout << std::fixed << std::setprecision(2);`

After this line of code, the output would only display 2 decimal places (like cents in dollars). The output might look something like:
`$29.99`
rather than
`$29.99865`
or something.

Similarly, I think the following code is for manipulating the outputs of boolean types so that it says "true/false" rather than "1/0". After the following code, all the boolean types that get printed will say True or False rather than just a binary representation:
std::cout << std::boolalpha;

Although, I don't know if this second example is actually within the `<iomanip> library, or just within `<streamio>`.

Long story short, I think so far we haven't been required to ever format stuff by the assignments, but your classmate wanted a specific "look" to their cout streams, so they used `<iomanip>` functions to help them get it.

**Comment by kartikayprashar** [comment]
@Cherry-Crane . @gavingro is correct. Use <Iomanip> to use the setprecision. Setprcision allows us to include decimal points in our numbers. Rather than saying $10, the setprecision(2) allows us to say $10.00.
